generator/src/pkg/model/deployment.go

```diff
@@ -110,16 +111,24 @@ type ContainerPortInstance struct {
 	ContainerPort int `yaml:"containerPort"`
 }
 
+type EnvInstance struct {
+	Name  string `yaml:"name"`
+	Value string `yaml:"value"`
+}
+
 type ContainerVolumeInstance struct {
 	MountPath string `yaml:"mountPath,omitempty"`
 	MountName string `yaml:"name,omitempty"`
 }
 
 type ReadinessProbeInstance struct {
 	HttpGet struct {
-		Path string `yaml:"path"`
-		Port int    `yaml:"port"`
-	} `yaml:"httpGet"`
+		Path string `yaml:"path,omitempty"`
+		Port int    `yaml:"port,omitempty"`
+	} `yaml:"httpGet,omitempty"`
+	Exec struct {
```

Comment:
Is this so you can probe readiness in grpc with your own implemented mechanism?

Reply:
yes